README.md

```diff
@@ -62,35 +75,39 @@ individual files in /usr/share/doc/*/copyright.
 Ubuntu comes with ABSOLUTELY NO WARRANTY, to the extent permitted by
 applicable law.
```

Comment:
we can remove all the banner stuff, it is confusing and does not give any useful information